moveit_simple/include/moveit_simple/robot.h

```diff
@@ -557,10 +583,10 @@ class Robot
   virtual std::vector<double> getJointState(void) const;
 
   bool getIK(const Eigen::Isometry3d pose, const std::vector<double> &seed,
-    std::vector<double> &joint_point, double timeout = 1) const;
+    std::vector<double> &joint_point, double timeout = 1, bool limit_joint_windup = false) const;
```

Comment:
If this is already a class variable you shouldn't need to pass it in here.

Reply:
@mlautman this was intended since it's crucial to allow `getIK()` calls with and without joint windup limitation even if the flag is set which is only intended to be used with the public member functions. `getIK()` is a private helper and is needed with limit_joint_windup enabled and disabled. For instance `getPickPlaceJointSolutions()` in #91 uses joint windup limitation for pick pose IK but then must call `getIK()` with limit_joint_windup=false so that the object orientation is not invalidated.